picoscope/ps4000a.py

```diff
@@ -313,11 +358,12 @@ def _lowLevelSetDataBuffer(self, channel, data, downSampleMode,
         segmentIndex is unused, but required by other versions of the API
         (eg PS5000a)
         """
+        dataPtr = data.ctypes.data_as(POINTER(c_int16))
```

Comment:
why is this change necessary?

Reply:
Similar to before, the ps4000 library didn't do this properly so I mostly copied the implementation from ps5000a. Some of the variable names were changed along the way, but it became more consistent with a newer version.

Reply:
Thanks!